src/coreclr/jit/earlyprop.cpp

```diff
@@ -4,7 +4,7 @@
 //                                    Early Value Propagation
 //
 // This phase performs an SSA-based value propagation optimization that currently only applies to array
-// lengths, runtime type handles, and explicit null checks. An SSA-based backwards tracking of local variables
+// lengths and explicit null checks. An SSA-based backwards tracking of local variables
 // is performed at each point of interest, e.g., an array length reference site, a method table reference site, or
```

Comment:
"a method table reference site," => also no longer applicable?

Reply:
Right. Will fix that with my next PR.